pkg/cmd/outputs.go

```diff
@@ -47,7 +47,7 @@ func NewSetOutputCommand() *cobra.Command {
 				return err
 			}
 
-			fmt.Fprintln(cmd.OutOrStdout(), "Successfully set output.")
+			fmt.Fprintln(cmd.OutOrStdout(), `Successfully set output for "`+key+`".`)
```

Comment:
You can get quotes with escaping automatically in Go using `strconv.Quote(string)` or using the `%q` modifier to `Printf`-style functions (you'd have to change this to `fmt.Fprintf` and put a newline at the end). Other than that, this looks good, thank you!